Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
@dunn if you installed OpenSSL to the custom location, you can use environment variable `OPENSSL` to specify the path to it. But I assume this is a wrong way for Homebrew. So I wonder why the `--qt-config` didn't work. I'll check that ASAP.

@mikemcquaid @DomT4 now when we have QtBase as a git submodule we can pull down updates pretty easy and fast. When Qt 5.6 released (with SecureTransport support) we will update our copy too.